gpMgmt/test/behave/mgmt_utils/gpexpand.feature

```diff
@@ -660,3 +660,33 @@ Feature: expand the cluster by adding more segments
         And verify that the path "gpperfmon/logs" in each segment data directory does not exist
         And verify that the path "promote" in each segment data directory does not exist
         And verify that the path "db_analyze" in each segment data directory does not exist
+
+    @gpexpand_no_mirrors
+    @gpexpand_segment
+    Scenario: gpexpand should skip already expanded/broken tables when redistributing
+        Given the database is not running
+        # need to remove this log because otherwise SCAN_LOG may pick up a previous error/warning in the log
+        And the user runs command "rm -rf ~/gpAdminLogs/gpinitsystem*"
```

Comment:
I think, these changes also should be described in commit message

Reply:
This line presents in all tests above, so it should have been there from the start, I don't think it has to be described

Reply:
Done